src/openstudio_lib/OSDropZone.hpp

```diff
@@ -126,7 +135,7 @@ class OSDropZone2
   boost::optional<OtherModelObjects> m_otherObjects;
   boost::optional<model::ModelObject> m_modelObject;
   boost::optional<model::ModelObject> m_getterResult;
-  std::vector<model::ModelObject> m_otherModelObjects;
+  std::set<model::ModelObject, ModelObjectHandleLess> m_otherModelObjects;
```

Comment:
Were you forced to specify a custom comparator?

Reply:
I just wanted to make sure that we were using the handle as the set key, I wasn't sure what `std::set<IdfObject>` would use as the key on it's own.  Using a set just simplified some of the code inside for adding or removing objects to watch (we only want to watch each other object once).